istioctl/cmd/waypoint.go

```diff
@@ -102,6 +102,12 @@ func waypointCmd() *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("failed to create Kubernetes client: %v", err)
 			}
+			if _, err = client.Ext().ApiextensionsV1().CustomResourceDefinitions().Get(context.Background(),
+				"gateways.gateway.networking.k8s.io", metav1.GetOptions{}); err != nil {
```

Comment:
Can you use this variable? https://github.com/istio/istio/blob/master/pkg/config/schema/gvk/resources.gen.go#L22